docs/requirements.txt

```diff
@@ -10,3 +10,4 @@ sphinx_rtd_theme==0.4.3
 sphinxcontrib.spelling==5.0.0
 sphinxcontrib-imagehelper==1.1.1
 vcstool==0.2.7
+typing-extensions>=3.6.4
```

Comment:
Dependency was required in my local environment for some reason.

Reply:
I do not think it wise to change the dependencies. Maybe this is caused because it is being run in Ubuntu 22.04 instead of Ubuntu 20.04. If Read the Docs has issues generating the documentation, then we can look into it (I am afraid that the fix we introduced the other day in every documentation repo is not in 2.1.x)

Reply:
This was tested on Ubuntu 20.04 but I agree, I'll revert this change.